### DISCUSSION TITLE
Can I print e.g. data.frames to the debug console without wrapping in `print`

### DISCUSSION THREAD

**Discussion by jayhaast** [ManuelHentschel/vscDebugger#191] [Q&A] [answered]
This is an awesome package thank you developers!

The only thing I find clunky about it is that if I want to view large objects (long vectors, data.frames etc.) in the debug console I need to wrap this in `print`. 

E.g. if I call `mtcars` from the debugger I get a printout `'data.frame':	6 obs. of  11 variables:`. Is there an option that matches what the standard R console spits out - i.e. the actual data frame in this example?

I have tried fiddling with the `r.debugger.printStdout` and `r.debugger.printSinkSocket` options but they don't seem to do what I'm after.

Thanks in advance

Jay

**Comment by ManuelHentschel** [comment, marked as answer]
Does `options(vsc.printArrays = TRUE)` work for you?

**Reply by jayhaast** [reply]
It does, thank you!
